core/src/main/java/org/apache/iceberg/view/ViewMetadata.java

```diff
@@ -68,21 +61,27 @@ default Integer currentSchemaId() {
     return currentSchemaId;
   }
 
+  @Value.Parameter(order = 1001)
+  String uuid();
+
+  @Value.Parameter(order = 1002)
+  int formatVersion();
+
```

Comment:
Question, can we put Override and Value.parameter here, and remove it from TableMetadata?  It will be more clear.

Reply:
Could you please add more context? i couldn't understand this part.

Reply:
Yea exactly, will take be able to remove @Value.Parameter in the BaseMetadata, and keep the annotation all in the ViewMetadata.

Reply:
@nk1506 let me know if the question makes more sense.

Reply:
@szehon-ho , Since we are planning to keep common members with ViewMetadata too with `@Override`. Do we have a need of keeping `@Value.Parameter` in ViewMetadata. I think only override should works. 
WDYT ? I have updated the patch with only `@Override` and removed `@Value.Parameter` from everywhere.